tests/unit/src/app/check/controllers/national-insurance-number.test.js

```diff
@@ -76,6 +76,37 @@ describe("national insurance number", () => {
       });
     });
 
+    describe("with 200 status and requestRetry", () => {
+      it('should set "showRetryErrorSummary" to true', async () => {
+        req.axios.post = jest
+          .fn()
+          .mockResolvedValue({ status: 200, data: { requestRetry: true } });
+
+        await controller.saveValues(req, res, next);
+
+        expect(req.session.redirectToRetry).toBeTruthy();
+        expect(controller.hasRedirectToRetryShowing(req)).toBeTruthy();
```

Comment:
I think we should be stricter here, since you are actually setting this in the controller to be `true` should only use `toBeTruthy()` is if you have a situation where an expression evaluates to  `1, 'non-empty string', {}, [],  true`which in javascript are true like values for instance assuming 

```
       if (response.status === 200 && response.data.requestRetry === true) {
          req.session.redirectToRetry = redirectValue;
        }
```
Maybe `redirectValue` is such that it could be evaluated and results in a truthy value i.e. `1, 'non-empty string', {}, [],  true`

Reply:
This isn't a thing in JavaScript from the looks of it. The other tests also use toBeTruthy/toBeFalsey

Reply:
It would be `.toBe(true)`